package.json

```diff
@@ -29,44 +29,44 @@
     }
   },
   "dependencies": {
-    "@angular/animations": "~11.0.7",
-    "@angular/common": "~11.0.7",
-    "@angular/compiler": "~11.0.7",
-    "@angular/core": "~11.0.7",
-    "@angular/forms": "~11.0.7",
-    "@angular/platform-browser": "~11.0.7",
-    "@angular/platform-browser-dynamic": "~11.0.7",
-    "@angular/router": "~11.0.7",
-    "@angular/service-worker": "~11.0.7",
+    "@angular/animations": "~14.0.0",
```

Comment:
This also updates ngqp-demo. What about the issues with Webpack? For me this doesn't run locally.

Reply:
That is because you use `raw-loader` for load ts files as string for examples. That raw-loader is removed from webpack 5 and angular cli 14 uses webpack 5

Reply:
I'm well aware of the problem, but I don't understand the change. If this simply doesn't work we of course can't merge it.